formtools/templates/formtools/wizard/wizard_form.html

```diff
@@ -1,4 +1,5 @@
 {% load i18n %}
+<form method="post" enctype="multipart/form-data">
```

Comment:
This will tag will need to be closed at the end of this template, too. `</form>`